marimo/_ast/app.py

```diff
@@ -232,13 +234,11 @@ class CellManager:
     def __init__(self) -> None:
         self._cell_data: dict[CellId_t, CellData] = {}
         self._cell_data = OrderedDict()
-        self._cell_id_counter = 0
         self.unparsable = False
 
     def create_cell_id(self) -> CellId_t:
-        cell_id = str(self._cell_id_counter)
-        self._cell_id_counter += 1
-        return cell_id
+        # 4 random letters
```

Comment:
I believe we need to set a random seed for instantiate requests that have object IDs/values to work. I suggest:

- create a `random.Random()` instance with a fixed seed in `App's` constructor: e.g., `self.rng = random.Random(42)`.
- use this instance (instead of the module-level `random`) in `create_cell_id`: `self.rng.choices(...)`

Reply:
sounds good, will do